ecl/dedicated_hypervisor/v1/_proxy.py

```diff
@@ -86,6 +86,12 @@ def create_server(self, name, networks, image, flavor,
             body["metadata"] = metadata
         return self._create(_server.Server, **body)
 
+    def update_server(self, name):
+
+        body = {}
+        body["name"] = name
+        return self._create(_server.Server, **body)
```

Comment:
```suggestion
        return self._update(_server.Server, **body)
```
誤記でしょうか。更新ですので updateです。